addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/environments-sc/parts/ScEnvSshConnRowExpanded.js

```diff
@@ -88,25 +99,73 @@ class ScEnvSshConnRowExpanded extends React.Component {
         <Table.Cell colSpan="3" className="p3">
           <Grid columns={2} stackable>
             <Grid.Row stretched>
-              <Grid.Column width={12}>{this.renderInfo()}</Grid.Column>
+              {this.isAppStreamEnabled ? (
+                <Grid.Column width={12}>{this.renderAppStreamInfo()}</Grid.Column>
+              ) : (
+                <Grid.Column width={12}>{this.renderInfo()}</Grid.Column>
+              )}
               <Grid.Column width={4}>
                 <Segment className="flex items-center">
-                  <div className="w-100 overflow-hidden">{this.renderCountDown()}</div>
+                  <div className="w-100">{this.renderCountDown()}</div>
                 </Segment>
               </Grid.Column>
             </Grid.Row>
           </Grid>
-          <div className="mt3">
-            Example:
-            <Segment className="mt2">
-              {example} <CopyToClipboard text={example} className="ml2 mt0" />
-            </Segment>
-          </div>
+          {this.isAppStreamEnabled ? (
+            <></>
+          ) : (
+            <div className="mt3">
+              Example:
+              <Segment className="mt2">
+                {example} <CopyToClipboard text={example} className="ml2 mt0" />
+              </Segment>
+            </div>
+          )}
         </Table.Cell>
       </Table.Row>
     );
   }
 
+  renderAppStreamInfo() {
+    const interfaces = this.networkInterfaces;
+    const moreThanOne = _.size(interfaces) > 1;
+
+    return (
+      <div>
+        <b>Connection instructions for your AppStream workspace:</b>
+        <List bulleted>
+          <List.Item>
+            The IP Address or DNS of the instance.{' '}
+            {moreThanOne ? 'Ask your administrator if you are not sure which one to use:' : ''}
+            <List>
+              {_.map(interfaces, network => (
+                <List.Item key={network.value} className="flex">
+                  {this.renderHostLabel(network)}
+                  <CopyToClipboard text={network.value} />
+                </List.Item>
+              ))}
+            </List>
+          </List.Item>
+          <List.Item>Click the &apos;Connect&apos; button below to navigate to the AppStream instance</List.Item>
+          <List.Item>You downloaded the private key when you created the SSH key.</List.Item>
+          <List.Item>Save this key&apos;s content into AppStream&apos;s Notepad application in .PEM format</List.Item>
```

Comment:
Do we need to add guidance on the directory they should use. Something like `C:\Users\Photon\Documents`. I think a bunch of dirs are admin only, so might be worth adding this info.

Even though the file would eventually be deleted, I suggest that we add a line for manual deletion once SSH is established

Reply:
As discussed, we'll be directing users to store their PEM files in Downloads and deleting once connected.